pages/common/pnpm.md

```diff
@@ -0,0 +1,40 @@
+# pnpm
+
+> Fast, disk space efficient package manager for Javascript and Node.js.
+> Manage Node.js projects and their module dependencies.
+> More information: <https://pnpm.io//>.
+
+- Interactively create a `package.json` file:
+
+`pnpm init`
+
+- Download all the packages listed as dependencies in package.json:
+
+`pnpm install`
+
+- Download a specific version of a package and add it to the list of dependencies in `package.json`:
+
+`pnpm install {{module_name}}@{{version}}`
+
+- Download a package and add it to the list of dev dependencies in `package.json`:
+
+`pnpm install {{module_name}} --save-dev`
+
+`pnpm install -D {{module_name}}`
+
+- Download a package and install it globally:
+
+`pnpm install -g {{module_name}}`
```

Comment:
Are you sure, this example is correct? I can't find it in <https://pnpm.io/cli/install>.

Reply:
yep it is, I've used it multiple times, also whenever pnpm updates, the command they suggest us is also this `pnpm install -g pnpm`

Reply:
Could we use `--global` here?  Or is it preferrable not to?